pyiso/caiso.py

```diff
@@ -283,7 +284,24 @@ def get_AS_dataframe(self, anc_region, latest=True, start_at=False, end_at=False
         MW columns holds $/MW float.
         If no data, returns an empty dataframe.
         """
-        # set args
+        # set args, handle latest differently than LMP b/c daily publication schedule
```

Comment:
Ah, the 20-min window problem makes sense. But having latest=True return a full day of data, especially a full day of data in the future, is super different from the behavior of latest=True on all other methods. Just trying to think of the simplest way to keep latest=True consistent while also useful...